cincinnati/src/plugins/internal/graph_builder/commons.rs

```diff
@@ -19,6 +19,7 @@ pub mod tests {
         )
     }
 
+    #[allow(dead_code)]
```

Comment:
f9076203267ac doesn't explain why we're allowing dead code here or elsewhere.  Can you add a note with motivation?  I'd expect there is some reason we can't just drop this dead code, and it would be good to have that available via `git blame ...`.

Reply:
This function is not being used now, and I don't think it has ever been. I guess its safe to be removed, but I'd rather mark it as dead code in this PR and remove it in the follow up

Reply:
@vrutkovs I would suggest that we can remove the code (this and others) completely. This is not a large function so we can add it back if we need it in future.